future-architect/vuls
# feat(GAdocker): Publish docker image with Github Actions

# What did you implement:

Publish docker image "vuls/vuls" with Github Actions when Vuls is added tag or pushed at master branch.

## Type of change


# How Has This Been Tested?

Tested on my repository is forked. 

1. Push to my repository (branch:GAdocker)
2. Merge GAdocker branch
3. Run [Github Actions](https://github.com/tttfrfr2/vuls/runs/3394424090?check_suite_focus=true) (can check how run Github Actions "Publish-docker-image")
4. Update docker image ([docker hub](https://hub.docker.com/layers/5338027/vuls/vuls/latest/images/sha256-c46934e23dc9d8eea45c8227b296a1f9f8ad368321811350c67742d349b929bb?context=repo)) (can check vuls:vuls was updated by tttfrfr2)

# Checklist:
You don't have to satisfy all of the following.


# Tips

Note that this Github Action "Publish-docker-image" is run correctly only if Secrets, "DOCKERHUB_USERNAME" and "DOCKERHUB_TOKEN", were setting.

***Is this ready for review?:*** Yes
